docs/pipeline/enrichments/available-enrichments/pii-pseudonymization-enrichment/index.md

```diff
@@ -42,9 +42,12 @@ It's **important** to keep these things in mind when using this enrichment:
 - Hashing a field can change its format (e.g. email) and its length, thus making a whole valid original event invalid if its schema is not compatible with the hashing.
 - When updating the `salt` after it has already been used, same original values hashed with previous and new salt will have different hashes, thus making a join impossible and/or creating duplicate values.
 
+### Anonymous mode
+Enrich 5.3.0 introduced anonymous mode. When [anonymousOnly](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow.enrichments/pii_enrichment_config/jsonschema/2-0-1#L155) is set to true, PII fields are masked only if the `SP-Anonymous` header is present.
```

Comment:
@mscwilson do we have a non-tracker-specific page about anonymous tracking?

Reply:
nope. Closest is this Recipe, I don't know how up to date it is https://docs.snowplow.io/docs/resources/recipes-tutorials/recipe-anonymous-tracking/

Reply:
no, I don't like that page. There's no good place for that info at the moment, so we can come back to it another time